packages/webapp/src/store/derive.ts

```diff
@@ -128,9 +127,8 @@ export function getOpponentData(gdata: FetchedGameData|null, player: Address|nul
 /**
  * @see module:store/read#getPrivateInfo
  */
-export function getPrivateInfo(gameID: bigint|null, player: Address|null): PrivateInfo|null {
+export function getPrivateInfo(gameID: bigint|null, player: Address|null, privateInfoStore: PrivateInfoStore): PrivateInfo|null {
```

Comment:
You will also need to change `getPrivateInfo` in `read.ts` to accommodate this. Let's get into the habit of running `make check` regularly. Will add an issue for adding github checks.

Reply:
Yep have this as a change, haven't committed as working on the animation change suggested as well.

Reply:
Fantastic, keep up the good work :)